src/parameterizations/lateral/MOM_Zanna_Bolton.F90

```diff
@@ -141,9 +149,19 @@ subroutine ZB2020_init(Time, G, GV, US, param_file, diag, CS, use_ZB2020)
                  "subgrid momentum parameterization of mesoscale eddies.", default=.false.)
   if (.not. use_ZB2020) return
 
+  call get_param(param_file, mdl, "ZB2020_USE_ANN", CS%use_ann, &
+                 "ANN inference of momentum fluxes", default=.false.)
+
+  call get_param(param_file, mdl, "ZB2020_ANN_STENCIL_SIZE", CS%stencil_size, &
+                 "ANN stencil size", default=3)
+
+  call get_param(param_file, mdl, "ZB2020_ANN_FILE_TALL", CS%ann_file_Tall, &
+                 "ANN parameters for prediction of Txy, Txx and Tyy netcdf input", &
+                 default="INPUT/EXP1/Tall.nc")
+
   call get_param(param_file, mdl, "ZB_SCALING", CS%amplitude, &
                  "The nondimensional scaling factor in ZB model, " //&
-                 "typically 0.5-2.5", units="nondim", default=0.5)
```

Comment:
The MOM6 consortium has a process for obtaining a consortium-wide consensus before changing default values for existing parameters, so that we do not inadvertently change other people's solutions.  Please restore the default value for `ZB_SCALING` to its previous value of 0.5, and I will add the proposal to change the default for `ZB_SCALING` to the 2026 list of proposed default changes that we will be considering at a consortium-wide meeting in about June of 2026.